frontend/src/components/Navbar/index.tsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import {
+  AppBar,
+  Typography,
+  Container,
+  IconButton,
+  Drawer,
+  List,
+  ListItem,
+} from "@material-ui/core";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import Toolbar from "@material-ui/core/Toolbar";
+import { Link } from "react-router-dom";
+import MenuIcon from "@material-ui/icons/Menu";
+import { linklist } from "./linklist";
```

Comment:
instead of using link list navlink should have been used to reduce code repeatation.

Reply:
`Navlinks` is a component and `linklist` is a list/array of links. `Navlinks` is used in the homepage to show links in a specific pattern(fixed position), so it cannot be reused in other components.